packages/client/src/containers/LandingPage/useFilteredTools.js

```diff
@@ -3,94 +3,100 @@ import getApiBaseUrl from '../../utils/getApiBaseURL';
 
 export function useFilteredTools() {
   const [tools, setTools] = useState([]);
-  const [baseUrl, setBaseUrl] = useState(`${getApiBaseUrl()}/api/tools`);
+  const [queryString, setQueryString] = useState('');
   const [isLoading, setIsLoading] = useState(true);
+  const [filters, setFilters] = useState({
+    category: [],
+    timeframe: [],
+    participantsNumber: [],
+  });
 
-  // fetching all tools
-  const getAllTools = useCallback(() => {
-    function fetchAllTools() {
-      setIsLoading(true);
-      setBaseUrl(`${getApiBaseUrl()}/api/tools`);
-
-      const promise = fetch(baseUrl).then((response) => response.json());
-      return promise;
+  useEffect(() => {
+    if (
+      filters.category === 0 &&
+      filters.participantsNumber === 0 &&
+      filters.timeframe === 0
+    ) {
+      setQueryString('');
+      return;
     }
-    fetchAllTools().then((data) => {
-      setIsLoading(false);
-    });
-  }, [baseUrl]);
+
+    const category = filters.category
+      .map((option) => `category[]=${option}`)
+      .join('&');
+    const participantsNumber = filters.participantsNumber
+      .map((option) => `participantsNumber[]=${option}`)
+      .join('&');
+    const timeframe = filters.timeframe
+      .map((option) => `timeframe[]=${option}`)
+      .join('&');
+
+    const newQueryString = [category, participantsNumber, timeframe]
+      .filter((option) => option !== '')
+      .join('&');
+    setQueryString(`?${newQueryString}`);
+  }, [filters]);
+
+  // Removing all already selected options that belongs to a showAll button
```

Comment:
Nitpick: These comments aren't needed anymore, the names explain enough and the code is very simple. 
Some comments in this file are even false, showing why I don't believe in comments :D